deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1313,9 +1313,9 @@ def get_fim_mode_param(mode, key='FIM_MODE'):
     metadata = {key.lower(): mode}
     if mode == 'scheduled':
         return {key: ''}, metadata
-    elif mode == 'realtime' and sys.platform != 'darwin':
+    elif mode == 'realtime' and sys.platform != 'darwin' and sys.platform != 'sunos5':
```

Comment:
We should create a list `_os_excluded_from_rt_wd` to avoid these kind of issue in the future. This check should be done using that list.